### DISCUSSION TITLE
Speed insights

### DISCUSSION THREAD

**Discussion by PiotrSierant** [vercel/next.js#67239] [Help]
### Summary

Hi,

I currently have a Pro plan, which includes 3 domains. I can't enable speed insights on any of them. Why?

![image](https://github.com/vercel/next.js/assets/94246053/10a6d604-5259-470e-b70e-27e0e8628e85)

**Comment by wwwdepot** [comment]
You will not get help here bro. Go to their website and look for help https://vercel.com/help
